### ISSUE TITLE
soundcloud.getSongInfo is not a function

### ISSUE THREAD

**Issue by alexni15** [Androz2091/discord-player#181]
hi, When I put a soundcloud link, the bot does not connect and I get error message in the logs
```javascript
(node:1354) UnhandledPromiseRejectionWarning: TypeError: soundcloud.getSongInfo is not a function
    at /home/bot/Djboris/node_modules/discord-player/src/Player.js:158:57
    at new Promise (<anonymous>)
    at Player._searchTracks (/home/bot/Djboris/node_modules/discord-player/src/Player.js:142:16)
    at Player.play (/home/bot/Djboris/node_modules/discord-player/src/Player.js:341:38)
    at Object.exports.run (/home/bot/Djboris/commands/play.js:90:19)
    at module.exports (/home/bot/Djboris/events/message.js:26:13)
    at Client.emit (events.js:315:20)
    at MessageCreateAction.handle (/home/bot/Djboris/node_modules/discord.js/src/client/actions/MessageCreate.js:31:14)
    at Object.module.exports [as MESSAGE_CREATE] (/home/bot/Djboris/node_modules/discord.js/src/client/websocket/handlers/MESSAGE_CREATE.js:4:32)
    at WebSocketManager.handlePacket (/home/bot/Djboris/node_modules/discord.js/src/client/websocket/WebSocketManager.js:384:31)
(node:1354) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
```

    @discordjs/opus: 0.3.3
    discord-player: 3.0.3
    discord.js: 12.4.1

**Comment by hands8142** [comment]
fix #182

**Comment by alexni15** [comment]
The problem seems to be fixed with the last commit on the master branch. Thanks for the work you do.
